base/Dockerfile

```diff
@@ -26,5 +26,5 @@ ENV VENV=$HOME/env/neurosci
 RUN python3 -m venv $VENV && python3 -m venv --system-site-packages $VENV
 
 RUN $VENV/bin/pip3 install --upgrade pip
-RUN $VENV/bin/pip3 install parameters quantities neo "django<1.9" django-tagging future hgapi gitpython sumatra nixio
+RUN $VENV/bin/pip3 install parameters quantities neo==0.7.1 "django<1.9" django-tagging future hgapi gitpython sumatra nixio
```

Comment:
Why the need to pin Neo to this specific version?

Reply:
To maximise future reproducibility of the environment. There's no guarantee neo 0.9 or later will work with pynn 0.9.4. Shall I update to neo 0.7.2 here and use 0.8 for pynn 0.9.5?

Any reason to install neo in base as opposed to simulation?

Reply:
I think it's a good idea, but then shouldn't we pin all the dependencies (quantities, etc.)?

neo is in base because the "analysis" image also needs it (to put it another way, there are no NeuralEnsemble images that don't need it, unless I'm mistaken)

Reply:
Hopefully for most the requirements.txt in PyNN etc. will be sufficient to choose an appropriate one. I just needed a specific neo here due to an issue here: https://github.com/NeuralEnsemble/neuralensemble-docker/issues/9#issuecomment-483764236.

Reply:
Understood. Please can you add a comment referring to that issue, so we remember why neo is being pinned in the Dockerfile?

Reply:
Just realised my previous comment was ambiguous - please can you add a comment _within the Dockerfile_ referring to that issue?

Reply:
Added.